felix/config/config_params.go

```diff
@@ -196,6 +196,7 @@ type Config struct {
 	BPFHostConntrackBypass             bool             `config:"bool;true"`
 	BPFEnforceRPF                      string           `config:"oneof(Disabled,Strict,Loose);Loose;non-zero"`
 	BPFPolicyDebugEnabled              bool             `config:"bool;true"`
+	BPFForceTrackPacketsFromIfaces     []string         `config:"string-slice;;die-on-fail"`
```

Comment:
Should we make the default useful, perhaps `docker+`?  Do we need `die-on-fail`; think the default behaviour might be good enough for this param.

Reply:
Implemented as suggested: make `docker+` the default and remove `die-on-fail`